### DISCUSSION TITLE
workspace prompt access does not work

### DISCUSSION THREAD

**Discussion by na-ga** [open-webui/open-webui#7903] [Issues]
In the open webui workspace, the prompt access settings are not working as expected.

I have changed the prompt access to Public to share with other users. However, other users cannot see it. 
If you are an administrator, you can see the user's prompts. 
Does the ability to share the prompt with regular users work correctly?

We are using the latest `0.4.8` version of open webui.

**Comment by na-ga** [comment]
I have the Prompt access control set to `Public`, but I am experiencing a behavior where other users cannot see it.
![貼り付けた画像_2024_12_17_10_55](https://github.com/user-attachments/assets/198cf71b-1f5c-470d-a7a5-ef55c5600f93)

**Comment by DCauldy** [comment]
I would really like to see this feature implemented as well
